ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivCreatePrivacyGroupTest.java

```diff
@@ -54,7 +55,7 @@
 
   private final Enclave enclave = mock(Enclave.class);
   private final PrivacyParameters privacyParameters = mock(PrivacyParameters.class);
-  private final PrivacyController privacyController = mock(PrivacyController.class);
+  private final PrivacyController privacyController = mock(DefaultPrivacyController.class);
```

Comment:
nit: can this not be a mock of the PrivacyController interface?

Reply:
done